GTG/gtk/browser/taskbrowser.ui

```diff
@@ -370,6 +393,40 @@
                     <property name="position">1</property>
                   </packing>
                 </child>
+                <child>
+                  <object class="GtkBox" id="quickadd_pane">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <child>
+                      <object class="GtkEntry" id="quickadd_field">
+                        <property name="visible">True</property>
+                        <property name="can_focus">True</property>
+                        <property name="has_focus">True</property>
+                        <property name="is_focus">True</property>
+                        <property name="can_default">True</property>
+                        <property name="invisible_char">●</property>
+                        <property name="placeholder-text">Quickly create or open tasks here</property>
```

Comment:
Good, I'll refactor the entire code for this and respond to a comment you posted at the end :blush: